jest.setup.js

```diff
@@ -0,0 +1,2 @@
+console.log = jest.fn();
+console.info = jest.fn();
```

Comment:
possibly controversial. i was having a tough time w/ `yarn test -- --watch`, and so many HPM logs clogging up the terminal. happy to drop this too.

Reply:
🙄 probably should have left it out :) it was mainly for me

Reply:
Nice. Always wondered how to get rid of those! :D